README.md

```diff
@@ -73,6 +73,10 @@ Download a release of the toolchain for your platform from [Github
 releases](https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/releases)
 and extract the archive into an arbitrary directory.
 
+### Pre-requisite to use toolchain in Windows 10 and Windows 11
```

Comment:
Replace with just Windows (without 10 and 11 - list of supported Windows versions should be elsewhere, not need to copy it here).

Reply:
Fixed in latest version of patch.

Reply:
Fixed in latest version of patch.

Reply:
Fixed in latest version of patch.